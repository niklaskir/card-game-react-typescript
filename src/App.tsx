import React from 'react';
import Container from './components/Container';
import DetailsSection from './components/DetailsSection';
import ControlsSection from './components/ControlsSection';
import OverviewSection from './components/OverviewSection';

const App = () => {
  return (
    <Container>
      <DetailsSection />
      <ControlsSection />
      <OverviewSection />
    </Container>
  );
};

export default App;
