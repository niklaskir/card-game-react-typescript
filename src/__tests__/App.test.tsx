import App from '../App';
import React from 'react';
import renderer from 'react-test-renderer';
import { RecoilRoot } from 'recoil';

jest.mock('../components/Container');
jest.mock('../components/ControlsSection');
jest.mock('../components/DetailsSection');
jest.mock('../components/OverviewSection');

describe('App', function () {
  it('should render correctly', () => {
    const app = renderer
      .create(
        <RecoilRoot>
          <App />
        </RecoilRoot>,
      )
      .toJSON();

    expect(app).toMatchSnapshot();
  });
});
