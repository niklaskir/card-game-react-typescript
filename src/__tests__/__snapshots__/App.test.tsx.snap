// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render correctly 1`] = `
<div
  id="container"
>
  <div
    id="details_section"
  />
  <div
    id="controls_section"
  />
  <div
    id="overview_section"
  />
</div>
`;
